package.json

```diff
@@ -38,7 +40,9 @@
     "gulp-zip": "^3.0.2",
     "run-sequence": "^1.1.4",
     "semistandard": "^7.0.5",
-    "tape": "^4.4.0"
+    "tape": "^4.4.0",
+    "tape-expect": "^1.2.0",
```

Comment:
I wanted to use the fancy `t.expect(response.Trains[0]).to.have.keys` so that when the fixture changes you don't have to update the tests. I think you're right that validating values is generally better.